connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorCheckpointConfig.java

```diff
@@ -166,6 +170,25 @@ Duration consumerPollTimeout() {
         return Duration.ofMillis(getLong(CONSUMER_POLL_TIMEOUT_MILLIS));
     }
 
+    public boolean validate() {
+        Boolean emitCheckpointsValue = this.getBoolean(EMIT_CHECKPOINTS_ENABLED);
+        Boolean syncGroupOffsetsValue = this.getBoolean(SYNC_GROUP_OFFSETS_ENABLED);
+
+        if (!emitCheckpointsValue && !syncGroupOffsetsValue) {
+            LOG.warn("MirrorCheckpointConnector can't run without both " + SYNC_GROUP_OFFSETS_ENABLED + ", " +
+                    EMIT_CHECKPOINTS_ENABLED + " set to false");
+            return false;
+        }
+
+        boolean requireOffsetSyncs = emitCheckpointsValue || syncGroupOffsetsValue;
+        if (!"true".equals(Optional.ofNullable(this.originals().get(EMIT_OFFSET_SYNCS_ENABLED)).orElse("true")) & requireOffsetSyncs) {
```

Comment:
Maybe we should use `EMIT_OFFSET_SYNCS_ENABLED_DEFAULT` here?

Reply:
I changed this to check if it is "false" directly instead of `!"true"` so no need to use `EMIT_OFFSET_SYNCS_ENABLED_DEFAULT` here